gitpod-init.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+gp env GEM_HOME=/workspace/.rvm
+gp env GEM_PATH=/workspace/.rvm
+gp env BUNDLE_PATH=/workspace/.rvm
+eval $(gp env -e)
```

Comment:
Loads the `gp env` set environment variables so we have access to the proper gem location.